bin/sl-nginx-ctl.js

```diff
@@ -24,15 +23,19 @@ var parser = new Parser([
     ':v(version)',
     'h(help)',
     'b:(base)',
-    'c:(control)',
-    'l:(listen)',
+    'A:(api)',
+    'n(no-api)',
+    'C:(control)',
+    'R:(routable-addr)',
```

Comment:
What about `-L` for where to listen for the API? That is basically what PM uses (though pm doesn't take a URL, which it probably should).